e2e_test/source/cdc_inline/sql_server_cdc/sql_server_cdc.slt

```diff
@@ -254,11 +254,17 @@ CREATE TABLE shared_orders (
 )  from mssql_source table 'dbo.orders';
 
 statement ok
-CREATE TABLE shared_single_type (
-    id INT,
-    c_time time,
-    PRIMARY KEY (id)
-) from mssql_source table 'dbo.single_type';
+CREATE TABLE shared_single_type (*) from mssql_source table 'dbo.single_type';
+
+statement ok
+CREATE TABLE shared_sqlserver_all_data_types (
+    *
+) from mssql_source table 'dbo.sqlserver_all_data_types';
+
+sleep 5s
```

Comment:
What's the intention of the sleep here? create - sleep - drop looks a little weird.

Reply:
Previously, I found if the types found by `(*)` are wrongly converted to RW types, RW will crash explicitly when consuming the data in the upstream table. I've fixed the found mismatch in this PR. This test `sleep 5s` is to ensure  the data is consumed and the mismatch is encountered so that it can be exposed.